spec/requests/events_request_spec.rb

```diff
@@ -100,4 +104,66 @@
       expect(response).to have_http_status(400)
     end
   end
+
+  describe 'PATCH abort' do
```

Comment:
You need to add a few specs. 

1. Ensure bad request is returned (and more importantly no event is mutated) when event id doesn't exist
2. Same as 1 but with bridge id
3. Valid event but user doesn't own the resource
4. Valid bridge but user doesn't own the resource

Reply:
We need to add 3 & 4 to whatever controllers you updated yesterday. I believe it was headers & env vars. This can be done in a separate pr though.